ablog/start.py

```diff
@@ -329,9 +329,6 @@
 # typographically correct entities.
 #html_use_smartypants = True
 
-# Custom sidebar templates, maps document names to template names.
-#html_sidebars = {}
```

Comment:
What was the reason for this removal?

Reply:
@nabobalis There's a duplicate entry up about 50 lines with a different set of defaults. So if you use this one, it supercedes the other one. The other one had a better set of defaults and was arguably in in a better location, so I just deleted this one.